drjt/Test-My-List-Util/lib/Test/My/List/Util.pm

```diff
@@ -0,0 +1,153 @@
+package Test::My::List::Util;
+
+use 5.010;
+use strict;
+use warnings;
+
+use Test::Builder;
+my $Test = new Test::Builder;
+
+
+use Exporter qw(import);
+our @EXPORT = qw( sum_ok );
+
+=head1 NAME
+
+Test::My::List::Util - The great new Test::My::List::Util!
+
+=head1 VERSION
+
+Version 0.01
+
+=cut
+
+our $VERSION = '0.01';
+
+=head1 SYNOPSIS
+
+Quick summary of what the module does.
+
+Perhaps a little code snippet.
+
+    use Test::My::List::Util;
+
+    my $foo = Test::My::List::Util->new();
+    ...
+
+=head1 EXPORT
+
+A list of functions that can be exported.  You can delete this section
+if you don't export anything, such as for a purely object-oriented module.
+
+=head1 SUBROUTINES/METHODS
+
+=head2 sum_ok
+
+The only sum-specific thing about this is the diagnostic message.
+le sigh.
+
+=cut
+
+sub sum_ok {
+  my ( $actual, $expected, $name ) = @_;
+
+  if ( $actual == $expected ) {
```

Comment:
You probably want to make sure your parameters _can_ be compared numerically, to make what would be warnings a bit more explicit.